examples/todomvc.rs

```diff
@@ -1,10 +1,13 @@
-#![allow(non_snake_case)]
+#![warn(clippy::pedantic)]
+#![allow(clippy::must_use_candidate)]
```

Comment:
Is it worth adding pedantic clippy lints in examples?

IMO I'd rather just keep the examples clean and straightforward. Not worried about warnings.

Reply:
> IMO I'd rather just keep the examples clean and straightforward.

That's why I added extra warnings, to make sure the code is clean. No offense, but I don't think two lines of Clippy make the code any less straightforward.